models/handpose_estimation_mediapipe/README.md

```diff
@@ -30,3 +33,5 @@ All files in this directory are licensed under [Apache 2.0 License](./LICENSE).
 ## Reference
 
 - MediaPipe Handpose: https://github.com/tensorflow/tfjs-models/tree/master/handpose
+- MediaPipe hands model and model card: https://google.github.io/mediapipe/solutions/models.html#hands
```

Comment:
What is model card?

Reply:
Model card is a document of MediaPipe model. There are some details about this model.